docs/resources/vba-differences.md

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Differences between Office Scripts and VBA macros'
+description: 'The behavior and API differences between Office Scripts and Excel VBA macros.'
+ms.date: 03/20/2020
+localization_priority: Normal
+---
+
+# Differences between Office Scripts and VBA macros
+
+Office Scripts and VBA macros have a lot in common. They both allow users to automate solutions through an easy-to-use action recorder and allow edits of those recordings. Both frameworks are designed to empower people who may not consider themselves programmers to create small programs in Excel.
+The fundamental difference is that Office Scripts are designed for Excel on the web and VBA macros are designed for desktop solutions.
```

Comment:
OS are really based on a cross-plat design with security first, so they miss out on some capabilities but add others. We don't currently have them in win32, but I don't think we should imply we won't in the future

Reply:
Reworded to make that distinction and remove the implication this will be forever web-only.

As to your second point, is that adequately covered in the "Platform and ecosystem" section?

Reply:
I think so ... I made some comments before I read stuff after.